maven/jvm.bzl

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (C) 2018 Square, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+
+# Description:
+#   A custom import rule that doesn't return an ijar, instead returning the raw .jar file as
```

Comment:
I imagine this is a fork of something, maybe provide a link to the original?

Reply:
Done.